src/Transform/ONNX/ConstProp.cpp

```diff
@@ -503,9 +503,26 @@ class ConstPropSplitPattern : public OpRewritePattern<ONNXSplitOp> {
 
     // Rewrite
     unsigned outputNum = splitOp->getNumResults();
-    int64_t rank = splitOp->input().getType().cast<ShapedType>().getRank();
+    Value splitInput = splitOp->input();
+    int64_t rank = splitInput.getType().cast<ShapedType>().getRank();
     IntegerAttr axisAttr = splitOp->axisAttr();
     ArrayAttr splitAttr = splitOp->splitAttr();
+    if (!splitAttr) {
+      // If split parameter is not specified, it is constructed from output.
```

Comment:
I guess you mean "... constructed from the input".

Reply:
I forgot to update.. Thanks!